04-One-Dimensional-Kalman-Filters.ipynb

```diff
@@ -472,7 +472,11 @@
     "    posterior = gaussian_multiply(likelihood, prior)\n",
     "    return posterior\n",
     "\n",
-    "update(pos, move)"
```

Comment:
This is not intuitive. The update function should be used to update the same type of state.
E.g. we should update (position and position) not (position and displacement)!

Reply:
Sure, but we can't just delete the update call, it needs to be called with a measurement. I cannot accept this pull like this.

Reply:
Sorry, github breaks things apart, I didn't see that you replaced the update with the correct call. Everything looks great.